chip/gpio.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+
+from os import uname
+from socket import gethostname
+from time import sleep
+import sys
+import cloud4rpi
+import chip
+
+# Get the GPIO module here: https://github.com/xtacocorex/CHIP_IO
+import CHIP_IO.GPIO as GPIO  # pylint: disable=F0401
```

Comment:
Maybe. I simply followed the examples here https://github.com/xtacocorex/CHIP_IO 
We should check that `from CHIP_IO import GPIO` works, I'm not absolutely sure that these statements are equivalent